code/game/objects/items/toys.dm

```diff
@@ -503,207 +503,324 @@
 	name = "Detective action figure"
 	desc = "A \"Space Life\" brand Detective action figure."
 	icon_state = "detective"
+	toysay = "This airlock has grey jumpsuit and insulated glove fibers on it."
 
 /obj/item/toy/figure/dsquad
 	name = "Space Commando action figure"
 	desc = "A \"Space Life\" brand Space Commando action figure."
 	icon_state = "dsquad"
+	toysay = "Eliminate all threats!"
 
 /obj/item/toy/figure/engineer
 	name = "Engineer action figure"
 	desc = "A \"Space Life\" brand Engineer action figure."
 	icon_state = "engineer"
+	toysay = "Oh god, the tesla is loose!"
 
 /obj/item/toy/figure/geneticist
 	name = "Geneticist action figure"
 	desc = "A \"Space Life\" brand Geneticist action figure, which was recently dicontinued."
 	icon_state = "geneticist"
+	toysay = "I'm not qualified for this job."
 
 /obj/item/toy/figure/hop
 	name = "Head of Personnel action figure"
 	desc = "A \"Space Life\" brand Head of Personnel action figure."
 	icon_state = "hop"
+	toysay = "Giving out all access!"
 
 /obj/item/toy/figure/hos
 	name = "Head of Security action figure"
 	desc = "A \"Space Life\" brand Head of Security action figure."
 	icon_state = "hos"
+	toysay = "I'm here to win, anything else is secondary."
 
 /obj/item/toy/figure/qm
 	name = "Quartermaster action figure"
 	desc = "A \"Space Life\" brand Quartermaster action figure."
 	icon_state = "qm"
+	toysay = "Hail Cargonia!"
 
 /obj/item/toy/figure/janitor
 	name = "Janitor action figure"
 	desc = "A \"Space Life\" brand Janitor action figure."
 	icon_state = "janitor"
+	toysay = "Look at the signs, you idiot."
 
 /obj/item/toy/figure/agent
 	name = "Internal Affairs Agent action figure"
 	desc = "A \"Space Life\" brand Internal Affairs Agent action figure."
 	icon_state = "agent"
+	toysay = "Standard Operating Procedure says they're guilty! Hacking is proof they're an Enemy of the Corporation!"
 
 /obj/item/toy/figure/librarian
 	name = "Librarian action figure"
 	desc = "A \"Space Life\" brand Librarian action figure."
 	icon_state = "librarian"
+	toysay = "One day while..."
 
 /obj/item/toy/figure/md
 	name = "Medical Doctor action figure"
 	desc = "A \"Space Life\" brand Medical Doctor action figure."
 	icon_state = "md"
+	toysay = "The patient is already dead!"
 
 /obj/item/toy/figure/mime
 	name = "Mime action figure"
 	desc = "A \"Space Life\" brand Mime action figure."
 	icon_state = "mime"
+	toysay = "..."
 
 /obj/item/toy/figure/miner
 	name = "Shaft Miner action figure"
 	desc = "A \"Space Life\" brand Shaft Miner action figure."
 	icon_state = "miner"
+	toysay = "Oh god, it's eating my intestines!"
 
 /obj/item/toy/figure/ninja
 	name = "Space Ninja action figure"
 	desc = "A \"Space Life\" brand Space Ninja action figure."
 	icon_state = "ninja"
+	toysay = "Oh god! Stop shooting, I'm friendly!"
 
 /obj/item/toy/figure/wizard
 	name = "Wizard action figure"
 	desc = "A \"Space Life\" brand Wizard action figure."
 	icon_state = "wizard"
+	toysay = "Ei Nath!"

```

Comment:
I don't think our Wizard, even the one replaced with Technomancer, had Ei Nath.

Reply:
Maybe there's just an in-universe TV show that has space wizards, who knows. They're action figures.